dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/CompanyManagerTest.kt

```diff
@@ -59,52 +62,16 @@ class CompanyManagerTest(
     @Test
     fun `search for them one by one by using their names`() {
         for (company in testCompanyList) {
-            val searchResponse = testCompanyManager.searchCompaniesAndGetApiModel(
-                CompanySearchFilter(
-                    searchString = company.companyName,
-                    onlyCompanyNames = true,
-                ),
+            val searchResponse = testCompanyManager.searchCompaniesByNameOrIdentifierAndGetApiModel(
+                searchString = company.companyName,
             )
             assertTrue(
-                searchResponse.any { it.companyInformation.companyName == company.companyName },
+                searchResponse.any { it.companyName == company.companyName },
                 "The posted company could not be retrieved by searching for its name.",
             )
         }
     }
 
-    private fun testThatSearchForCompanyIdentifierWorks(identifier: CompanyIdentifier) {
```

Comment:
Why has this test been deleted.  Isn't it still a valid case that someone searches for companies on the framework data search with an identifier?

Reply:
Was moved to the e2e tests for the company data controller. Reason was that as we only received companies for which at least one framework dataset was uploaded, we either would have to mock the Framework data controller in this test or rewrite it as an e2e test. We decided to do the latter.

Reply:
Ok